src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap';
+@import './utils/variables';
```

Comment:
Ensure that the `variables` file is correctly set up and contains all the necessary SCSS variables used across the project. This is crucial for maintaining consistency and avoiding errors related to undefined variables.